src/blog/lagom-1-5-0-M1.md

```diff
@@ -1,27 +1,47 @@
+
 ## Highlights
 
-- Updates Play to [2.7.0-M2](https://blog.playframework.com/play-2-7-0-m2-released/) (see also [2.7.0-M1](https://blog.playframework.com/play-2-7-0-milestone-1-released/)). Play 2.7.x improves failure handling in RDBMS failure scenarios and adopts [Akka's Coordinated Shutdown](https://doc.akka.io/docs/akka/current/actors.html#coordinated-shutdown). It also removes some deprecated APIs Lagom exposed transparently so you may need to review your code. Make sure to review upstream changes. As part of this update, the dependency to the Jackson library (for java JSON serialization) was bumped to version `2.9.6` which introduced some new default settings that may require you to review your serialization settings.
-- Start adoption of [Akka's Service Discovery](https://developer.lightbend.com/docs/akka-management/current/discovery.html) which will become a building block of Lagom's `ServiceLocator` and help standardize service discovery scenarios. This will simplify production settings to tune cluster bootstrap and other service discovery strategies and combine them.
-- Updates Alpakka Kafka (aka `reactive-kafka` and `akka-stream-kafka`) to version [0.22](https://github.com/akka/alpakka-kafka/releases).
-- Improves Development Mode settings so each service can choose what network interface to bind to (defaults to `0.0.0.0`), which is particularly useful in some OS or corporate environments.
-- Many documentation improvements, especially to remove deprecated APIs in our code examples.
+### Updates Play to 2.7.x
+
+Lagom 1.5.0-M2 bumps the Play version to [2.7.0-M2](https://blog.playframework.com/play-2-7-0-m2-released/) (see also [2.7.0-M1](https://blog.playframework.com/play-2-7-0-milestone-1-released/)). Play now exposes the [Akka Coordinated Shutdown](https://doc.akka.io/docs/akka/current/actors.html#coordinated-shutdown) instance as a first class citizen so you can register your shutdown code using it instead of using Play's `ApplicationLifecycle`.
+
+The version of Akka HTTP powering the default HTTP Backend for both Play and Lagom was bumped to [`10.1`](https://akka.io/blog/news/2018/03/08/akka-http-10.1.0-released).
+
+Upgrading to Play 2.7 may cause some errors on your code as some deprecated APIs were removed (please review the [Play 2.7 Migration Guide](https://github.com/playframework/playframework/blob/master/documentation/manual/releases/release27/migration27/Migration27.md) or the release announcements linked above). Play also updated the Jackson library (for java JSON serialization) to `2.9.6` which includes reviewed defaults so you may need to check your serializer settings. The changes in Jackson default settings may not cause compilation errors so we recommend you exercise your serializers in tests to detect any potential issue.
```

Comment:
> Play also updated the Jackson library (for java JSON serialization) to `2.9.6`

We're not putting other version numbers in backticks… I think either style is fine (since it could be considered a code fragment) but we should be consistent.